tests/rptest/tests/node_id_assignment_test.py

```diff
@@ -119,6 +122,33 @@ def test_rejoin_with_wrong_node_id(self):
                                  expect_fail=True)
 
 
+class NodeIdAssignmentParallel(RedpandaTest):
+    """
+    Test that adds several nodes at once, assigning each a unique node ID.
+    """
+    def __init__(self, test_context):
+        super(NodeIdAssignmentParallel,
+              self).__init__(test_context=test_context, num_brokers=4)
+
+    def setUp(self):
+        self.redpanda.start([self.redpanda.nodes[0]],
+                            auto_assign_node_id=True,
+                            omit_seeds_on_idx_one=False)
+
+    @cluster(num_nodes=4)
+    def test_assign_multiple_nodes(self):
+        self.redpanda.start(self.redpanda.nodes[1:],
+                            auto_assign_node_id=True,
+                            omit_seeds_on_idx_one=False)
```

Comment:
1. what purpose does the slightly different setUp serve in the test? that seems a bit subtle
2. should start take parallel=True? even tho it is the default value, this test depends on it being True

Reply:
1. in retrospect the multiple assignments at once is already being hit because the default behavior in most tests is to start up with a root node still,  and the rest of the nodes join as is. But the point here is explicitly start one node, and then add the rest in parallel in the test body
2. done

Reply:
>the multiple assignments at once is already being hit because the default behavior in most tests is to start up with a root node still, and the rest of the nodes join as is.

Actually scratch that, I forgot that we don't use automatic node ID assignment in tests by default yet. The second point is still true though, and I added a comment explaining.